client/src/components/ReviewTile.js

```diff
@@ -1,24 +1,109 @@
-import React from "react"
+import React, { useState, useEffect, setStatus } from "react"
+
+import NewReviewForm from "./NewReviewForm.js"
+import EditReviewForm from "./EditReviewForm.js"
 
 const ReviewTile = (props) => {
+  const review = props.review
+
   let upVotes = 0
   let downVotes = 0
-  
-  props.votes.forEach(vote => {
+
+  review.votes.forEach(vote => {
     if(vote.direction === "up") {
       upVotes++
     } else {
       downVotes++
     }
   })
 
-  return (
-    <div>
-      <p>Score: {props.score} out of 5</p>
-      <p>{props.body}</p>
-      <p>Upvotes: {upVotes}  Downvotes: {downVotes}</p>
-    </div>
-  )
+  const reviewId = review.id
+  const showId = review.showId
+ 
+  const handleDelete = async (event) => {
+    event.preventDefault()
+
+    try {
+      const response = await fetch(`/api/v1/shows/${showId}/reviews/${reviewId}`, {
+        method: "DELETE"
+      }).then(() => setStatus('Deleted') )
+    } catch(error) {
+      console.error(`Error in Fetch: ${error.message}`)
+    }
+    
+    props.getShow() 
+  }
+
+  const [canEdit, setCanEdit] = useState(false)
+
+  const handleEdit = (event) => {
+    event.preventDefault()
+    setCanEdit(!canEdit)
+  }
+
+  if(props.userId === review.userId) {
+    if(canEdit) {
+      return (
+        <div>
+          <p>Score: {review.score} out of 5</p>
+          <p>{review.body}</p>
+          <p>Upvotes: {upVotes}  Downvotes: {downVotes}</p>
```

Comment:
lines 47 to 50 are duplicated numerous times. This means that your ReviewTile is getting so beefy that it can be broken down into another component. Move these lines into a file such as `ReviewDisplay`, and then import that into `ReviewTile` and call as needed. Should trim down our lines of code in this file dramatically.